homework_1/zad1/solution/testCases_test.go

```diff
@@ -0,0 +1,34 @@
+package solution_test
+
+type testCase struct {
+	inputStart int
+	inputEnd int
+
+	expectedOutput []string
+	expectingError bool
+}
+
+func getTestCases() []testCase {
+	return []testCase {
+		{
+			inputStart: 1,
+			inputEnd: 10,
+
+			expectedOutput: []string{"1", "2", "Fizz", "4", "Buzz", "Fizz", "7", "8", "Fizz", "Buzz"},
+			expectingError: false,
+		},
+		{
+			inputStart: 10,
+			inputEnd: 15,
+
+			expectedOutput: []string{"Buzz", "11", "Fizz", "13", "14", "FizzBuzz"},
+			expectingError: false,
+		},
+		{
+			inputStart: 2,
+			inputEnd: 1,
+
+			expectingError: true,
+		},
+	}
+}
```

Comment:
I see in all files you're missing a final empty line, it is also a convention to leave the last line empty (it's not just a Golang convention, but rather "text file" convention in general).
I believe the reason is to have a nice output in case you `cat` the text file from terminal, but there are few other minor benefits as well.